### PR TITLE
[BULK] DocuTune - Fix build validation issues: docs-link-absolute (part 1)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: Contribute/git-github-fundamentals.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ As a contributor to Docs content, you will interact with multiple tools and proc
 
 Git is an open-source version control system. It facilitates this type of project collaboration through *distributed version control* of files that live in *repositories*. In essence, Git makes it possible to integrate streams of work done by multiple contributors over time, for a given repository.
 
-GitHub is a web-based hosting service for Git repositories, such as those used to store [docs.microsoft.com](https://docs.microsoft.com) content. For any project, GitHub hosts the main repository, from which contributors can make copies for their own work.
+GitHub is a web-based hosting service for Git repositories, such as those used to [store]() content. For any project, GitHub hosts the main repository, from which contributors can make copies for their own work.
```
Comment (brittmsantos) [inline Contribute/git-github-fundamentals.md:17]: I'd correct this one to `[Microsoft Docs](https://docs.microsoft.com)`, like similar edits in this PR.

**Comment by brittmsantos** [comment]
#docutune-wait

File: Contribute/git-github-fundamentals.md (hunk 1 above)
Comment (alexbuckgit) [reply Contribute/git-github-fundamentals.md]: Fixed

**Comment by alexbuckgit** [comment]
#docutune-ready

**Review by JasonWHowell — APPROVED** [review]
Changes seem correct

**Comment by JasonWHowell** [comment]
@jehchow can you merge? I don't think we need 4-5 reviews

**Review by jehchow — APPROVED** [review]

**Pull request merged**